python/baseline/vectorizers.py

```diff
@@ -80,6 +80,10 @@ def count(self, tokens):
         self.max_seen = max(self.max_seen, seen)
         return counter
 
+    def reset(self):
```

Comment:
Like mentioned above when adding the count method to the bert vectorizer it seems like it would make sense to add these methods to the `AbstractVectorizer`